contribs/gmf/src/services/permalink.js

```diff
@@ -3,6 +3,8 @@ goog.provide('gmf.Permalink');
 goog.require('gmf');
 goog.require('ngeo');
 goog.require('ngeo.AutoProjection');
+/** @suppress {extraRequire} */
+goog.require('gmf.ObjectEditingManager');
```

Comment:
:-) then where, If I add it in the gmf.js, it's a requirement for all the module...